examples/getting_started/3_Deep_dive_into_the_anatomy_of_quantum_circuits.ipynb

```diff
@@ -41,7 +41,6 @@
     "# magic word for producing visualizations in notebook\n",
```

Comment:
<p>Delete the commented out "# import logging" (since it's right there in the line below)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>